src/FindApprenticeshipTraining/SFA.DAS.FindApprenticeshipTraining/InnerApi/Responses/GetEmployerFeedbackItem.cs

```diff
@@ -1,11 +1,51 @@
-﻿using System.Collections.Generic;
+﻿
+using System.Collections.Generic;
+using System.Linq;
+
 
 namespace SFA.DAS.FindApprenticeshipTraining.InnerApi.Responses
 {
     public class GetEmployerFeedbackItem
     {
         public IEnumerable<GetEmployerFeedbackAttributeItem> FeedbackAttributes { get; set; }
         public IEnumerable<GetEmployerFeedbackRatingItem> FeedbackRatings { get; set; }
+
+        public static implicit operator GetEmployerFeedbackItem(GetEmployerFeedbackResponse item)
```

Comment:
I think it would be worth adding some unit test coverage for the implicit operator - just to prove the happy and unhappy paths.

Reply:
done